src/backend/common/DefaultMemoryManager.cpp

```diff
@@ -166,13 +166,15 @@ void *DefaultMemoryManager::alloc(bool user_lock, const unsigned ndims,
             }
 
             lock_guard_t lock(this->memory_mutex);
-            free_iter iter = current.free_map.find(alloc_bytes);
```

Comment:
This type alias should be removed from here
https://github.com/arrayfire/arrayfire/blob/3693e6a0ea5ec89a39f4699102db1967de3b923d/src/backend/common/DefaultMemoryManager.hpp#L42